services/static-webserver/client/source/resource/osparc/tours/tis_tours.json

```diff
@@ -10,7 +10,7 @@
         "event": "tap"
       },
       "anchorEl": "osparc-test-id=notificationsContainer",
-      "text": "By clicking on the Bell, you will you see notifications about what Studies, Credits and Organizations.",
+      "text": "By clicking on the Bell, you will you see notifications about what Studies, Credits and Organizations were shared with you.",
```

Comment:
By clicking on the Bell, you will see notifications about which Studies, Credits and Organizations have been shared with you.